### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by cp-pratik-k**
### Purpose
- Toggle user status
- Delete all members profile pic and space logo on space delete
- Fix logo stored on another path in create space
- Add option to remove user invitation
- minor ui changes
- Fix admin employee list

### Summary of Changes
- Toggle user status
- Delete all members profile pic and space logo on space delete
- Fix logo stored on another path in create space
- Add option to remove user invitation
- minor ui changes
- Fix admin employee list

### Test steps
 - Run flutter test
 - Run app
 
### Conformity
- [x] Followed [git guidelines](https://github.com/canopas/flutter-developer-roadmap/blob/main/GitGuideline.md) for creating commit messages and Pull Request guidelines.
- [x] Self-approved the PR - reviewed the PR as a reviewer and gave it self-approval if everything is ok. If not, made the required changes.
- [x] Ensured that the PR satisfies all specified requirements in the ticket, including bug fixes and new features.
- [x] Provided test steps, including steps to reproduce the issue or test the new functionality, ensuring other team members can verify the changes.
- [x] Added/Updated proper code comments to make it easy-to-understand for other developers.
- [x] Reused code (if the same code was written twice, made it common and reused it at both places).
- [x] Removed unused or commented code if not required.
- [x] Ensured proper Dart naming conventions were used for variables, classes, and methods.
- [x] Localized user-facing strings.
- [x] Included screenshots/videos of behavior changes: Provided visual evidence of any changes to UI or behavior for easier review and understanding in the PR description.
- [x] Implemented proper error handling: Ensured that the code anticipated and handled potential errors and edge cases gracefully.
- [x] Avoided introducing technical debt: If the PR introduces technical debt, created and linked appropriate tickets for future resolution.
- [x] Included relevant unit tests: Wrote unit tests that focused on testing behavior and functionality, rather than merely covering lines of code.
- [x] Ensured code was performant and scalable: Verified that the changes did not introduce performance issues or bottlenecks and could scale as needed.
- [x] Ensured comments were up-to-date and relevant to the code to describe complex logic and to add understanding for other developers.
- [x] Marked the PR as ready before submitting it for review.

### CODE REVIEW

File: lib/ui/admin/drawer_options/edit_space/bloc/edit_space_bloc.dart [hunk 1]
```diff
@@ -61,8 +61,8 @@ class EditSpaceBloc extends Bloc<EditSpaceEvent, EditSpaceState>
       DeleteSpaceEvent event, Emitter<EditSpaceState> emit) async {
     emit(state.copyWith(deleteWorkSpaceStatus: Status.loading));
     try {
-      await _spaceService.deleteSpace(
-         spaceId:  _userStateNotifier.currentSpace!.id, owners: _userStateNotifier.currentSpace!.ownerIds,uid: _userStateNotifier.employeeId);
+      await _spaceService.deleteSpace(spaceId:  _userStateNotifier.currentSpace!.id, owners: _userStateNotifier.currentSpace!.ownerIds,uid: _userStateNotifier.employeeId);
+      await _storageService.deleteStorageFolder("images/${_userStateNotifier.currentSpaceId}");
       await _userStateNotifier.removeEmployeeWithSpace();
```
Comment (cp-sneha-s) [inline lib/ui/admin/drawer_options/edit_space/bloc/edit_space_bloc.dart:66]: Add constant for storage path.

**Review by cp-sneha-s — APPROVED** [review]
